packages/components/infinite-scroll-area/src/infinite-scroll-area.tsx

```diff
@@ -54,17 +54,19 @@ export const InfiniteScrollArea = forwardRef<InfiniteScrollAreaProps, "div">(
       orientation: _orientation,
       ...props,
     })
-    const {
+    let {
       className,
       children,
-      finish,
+      disabled,
+      finish: finishProp,
```

Comment:
@hirotomoyamada 
All changes related to destructure have been applied except for this line, since I renamed the `isFinish` to `finish` in the hook, please let me know if this is not as expected

Reply:
That's fine. I think `finishProp` is correct because it is covered by the internal `finish`.